doc/theory/sh_basis.rst

```diff
@@ -161,6 +160,15 @@ are also taken into account when reconstructing a spherical function. These
 full bases can successfully reconstruct asymmetric signals as well as
 symmetric signals.
 
+NOTE: 
+The definition of spherical harmonics that DIPY utilizes does not match the one
+in Wikipedia and scipy. Instead, DIPY follows the dMRI literature conventions,
+like in ``descoteaux07`` and ``tournier07``.
+The code in DIPY also follows the following convention:
+Let the SH be noted as $Y_{l}^m$. Then, $l$ is referred to as either order or
+l_value(s), and $m$ is referred to as either phase factor or m_value(s).
+This decisions were made as a result of the PR in https://github.com/dipy/dipy/pull/3086
```

Comment:
Since this PR is cited there, I would also update the section "Main changes" (at the beginning of the PR) to represent the actual final changes. It would be less confusing if someone looks at it in the future.

Reply:
Thank you both for the suggestions :)

- @karanphil I just updated the beginning of the PR. It should reflect the current changesnow.
- @CHrlS98 thanks for the grammar check. I will leave it as _"These decisions..."_. Pushing it in the next commit.